operator/pkg/controllers/configconnector/configconnector_controller.go

```diff
@@ -373,38 +370,35 @@ func (r *ConfigConnectorReconciler) removeClusterModeOnlySharedComponents(ctx co
 	sc := &corev1.ServiceAccount{}
 	sc.Namespace = k8s.CNRMSystemNamespace
 	sc.Name = k8s.KCCControllerManagerComponent
-	if err := controllers.DeleteObject(ctx, c, sc); err != nil {
```

Comment:
Nit:This one is sort of debateable (like maybe we should wrap the error instead?), hopefully it doesn't come up _too_ often!